server/app-engine/static/graas.js

```diff
@@ -44,9 +45,12 @@ var testDow = null;
 var version = null;
 
 const UUID_NAME = 'lat_long_id';
+const VEHICLE_ID_COOKIE_NAME = 'vehicle_id';
+var vehicleIDCookie = null;
 const MSG_NO = 'msg_no';
-const MAX_AGE = 10 * 60 * 60 * 24 * 365;
-const MAX_LIFE = 1000 * 60 * 60 * 24 * 7;
+const MAX_AGE_SECS = 10 * 60 * 60 * 24 * 365;
```

Comment:
You're playing a somewhat dangerous game here as long as we don't have a linter. It'd be pretty easy to overlook an instance or two where the name needs to be changed. I understand your desire for clean code, and I'm not saying don't do it, just be aware.

Reply:
Thanks. I just double checked for other files using these variables. It's a helpful reminder, can be easy to forget that these vars could be referenced in other files.